taxjar/taxjar-magento2-extension
# Add TaxJar logo to admin page, adjust sort order

### Context
This change cleans up admin interface found in Admin > Stores > Configuration > Sales > Tax.

### Description
This PR moves TaxJar to the top of the configuration page and replaces the text "TaxJar Sales Tax Automation" with the TaxJar logo.  
![Screen Shot 2020-08-27 at 5 47 37 PM](https://user-images.githubusercontent.com/44789510/91872488-29fba180-ec35-11ea-85f9-743737bd34e6.png)

### Performance
n/a

### Testing

1. Login to Magento's admin and navigate to Admin > Stores > Configuration > Sales > Tax
2. Confirm the TaxJar logo is rendered and that TaxJar appears at the top of the page

#### Versions
